navigation/navigation-common/src/commonMain/kotlin/androidx/navigation/NavDeepLink.kt

```diff
@@ -120,15 +540,19 @@ public expect class NavDeepLink internal constructor(
          */
         public inline fun <reified T : Any> setUriPattern(
             basePath: String,
-            typeMap: Map<KType, NavType<*>> = emptyMap(),
```

Comment:
Why was default arg removed?

Reply:
oh, you are right. expect/actual refactoring erased it

Reply:
done